console/EAP7-837_Customize_Management_Console_Title.adoc

```diff
@@ -0,0 +1,54 @@
+= Expose EJB runtime data / statistics
+:author:            Harald Pehl
+:email:             hpehl@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+Customise the title of the management console. A custom title can be specified in the user settings of the console. The title can contain the following placeholders:
+
+- `%n`: The `name` attribute of the the root resource
+- `%o`: The `organization` (standalone) resp. `domain-organization` attribute of the the root resource
+
+If the title is not specified, the built in title will be used. If a placeholder is used, but the attribute is undefined, the built in title will be used.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-837[EAP7-837]
+
+=== Dev Contacts
+
+* mailto:hpehl@redhat.com[Harald Pehl]
+
+=== QE Contacts
+
+* mailto:pjelinek@redhat.com[Pavel Jelinek]
+
+=== Affected Projects or Components
+
+* WildFly
+* HAL
+
+== Requirements
+
+- Add an entry in the user setting to customise the title
+- If no title is specified the built in title will be used
+- If a title is specified use this title as borwser / tab title
+- Replace `%n` with the value of the `name` attribute of the root resource
+- Replace `%o` with the value of the `organization` (standalone) resp. `domain-organization` (domain) attribute of the root resource
+- If a placeholder resolves to an undefined attribute use the built in title.
```

Comment:
Won't the user just think, that the feature doesn't work when the title won't change?

Reply:
Good point. First I thought about having "undefined" or "n/a" in the title. But I think a valid title is more professional. There's an error logged in the browser console what placeholder could not be evaluated.